sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -26,6 +27,15 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         this._swapView(feedsView);
     },
 
+    browseFeeds: function() {
+        var browseView = new Sourcemash.Views.BrowseView({
+            collection: new Sourcemash.Collections.Feeds(),
```

Comment:
Sounds like this calls for debugging

-- 
Alex Gerstein
gerstein.alex@gmail.com

On March 7, 2015 at 4:55:39 PM, Scott Gladstone (notifications@github.com) wrote:

In sourcemash/frontend/static/js/routers/app_router.js:

> @@ -26,6 +27,15 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
>          this._swapView(feedsView);
>      },
>   
> +    browseFeeds: function() {
> +        var browseView = new Sourcemash.Views.BrowseView({
> +            collection: new Sourcemash.Collections.Feeds(),
Tried that. The feeds page stopped rendering everything. Just blank.
—
Reply to this email directly or view it on GitHub.